mito-ai/mito_ai/utils/mito_server_utils.py

```diff
@@ -88,5 +89,147 @@ async def get_response_from_mito_server(
             pass
         
         http_client.close()
+
+
+async def stream_response_from_mito_server(
+    url: str,
+    headers: Dict[str, str],
+    data: Dict[str, Any],
+    timeout: int,
+    max_retries: int,
+    message_type: MessageType,
+    provider_name: str = "Mito Server",
+    reply_fn: Optional[Callable[[Union[CompletionReply, CompletionStreamChunk]], None]] = None,
+    message_id: Optional[str] = None,
```

Comment:
Is there any reason that these should be Optional? seems like they are pretty required! That's where the bug came from.